content/news/025/index.md

```diff
@@ -67,6 +67,34 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Pixie Wrangler]
+
+[![Screenshot of Pixie Wrangler showing pixies traveling along paths drawn
+by the player.](pixie_wrangler.png)][pw-itch]
+_Click the image to play the game in your desktop browser!_
+
+Pixie Wrangler ([itch.io][pw-itch], [GitHub][pw-github]) by [@rparrett]
+is a puzzle game reminiscent of old school printed circuit board design
+software.
+
+Help the Pixies get from their outputs to their inputs while doing battle
```

Comment:
Should this be 'from their inputs to their outputs'?

Reply:
Nah. In the game, they come out of the node labeled "output" and go into the node labeled "input."

But if that's not sounding good, I'm open to changing it up.

Reply:
Ah, okay - wasn't sure if it was a typo, but that makes sense :)